packages/react-router-dom/index.tsx

```diff
@@ -349,6 +350,20 @@ if (__DEV__) {
 
 export { ViewTransitionContext as UNSAFE_ViewTransitionContext };
 
+// TODO: (v7) Change the useFetcher data from `any` to `unknown`
+type FetchersContextObject = {
+  data: Map<string, any>;
+  register: (key: string) => void;
+  unregister: (key: string) => void;
+};
```

Comment:
This context holds the fetcher data handed off from the router and exposes it for `useFetcher`